pkg/manifest/manifest.go

```diff
@@ -24,7 +24,7 @@ import (
 const invalidStepErrorFormat = "validation of action \"%s\" failed"
 
 type Manifest struct {
-	// ManifestPath is the location from which the manifest was loaded, such as the path on the filesystem or a url.
+	// ManifestPath is location to the original, user-supplied manifest, such as the path on the filesystem or a url
```

Comment:
Oh nice, looks like you didn't end up needing to track the generated manifest path separately. 👍